assets/src/edit-story/app/media/local/test/useContextValueProvider.js

```diff
@@ -0,0 +1,263 @@
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-996x2048.jpg',
+      },
+      'web-stories-poster-portrait': {
+        file: 'IMAGE-640x853.jpg',
+        width: 640,
+        height: 853,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-640x853.jpg',
+      },
+      'web-stories-poster-landscape': {
+        file: 'IMAGE-853x640.jpg',
+        width: 853,
+        height: 640,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-853x640.jpg',
+      },
+      'web-stories-poster-square': {
+        file: 'IMAGE-640x640.jpg',
+        width: 640,
+        height: 640,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-640x640.jpg',
+      },
+      'web-stories-publisher-logo': {
+        file: 'IMAGE-96x96.jpg',
+        width: 96,
+        height: 96,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-96x96.jpg',
+      },
+      'web-stories-thumbnail': {
+        file: 'IMAGE-150x308.jpg',
+        width: 150,
+        height: 308,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-150x308.jpg',
+      },
+      full: {
+        file: 'IMAGE.jpg',
+        width: 1080,
+        height: 2220,
+        mime_type: 'image/jpeg',
+        source_url: 'http://wp.local/wp-content/uploads/2020/09/IMAGE.jpg',
+      },
+    },
+    attribution: undefined,
+  },
+];
+
+describe('useContextValueProvider', () => {
+  let reducerState;
+  let reducerActions;
+  beforeEach(() => {
+    reducerState = {
+      processing: [],
+      processed: [],
+      media: [],
+      pageToken: '',
+      mediaType: '',
+      searchTerm: '',
+    };
+    reducerActions = {
+      fetchMediaStart: jest.fn(),
+      fetchMediaSuccess: jest.fn(),
+      fetchMediaError: jest.fn(),
+      resetFilters: jest.fn(),
+      setMedia: jest.fn(),
+      setMediaType: jest.fn(),
+      setSearchTerm: jest.fn(),
+      setNextPage: jest.fn(),
+      setProcessing: jest.fn(),
+      removeProcessing: jest.fn(),
+      updateMediaElement: jest.fn(),
+      deleteMediaElement: jest.fn(),
+    };
+
+    const useUploadMediaResult = {
+      uploadMedia: jest.fn(),
+      isUploading: jest.fn(),
+    };
+    useUploadMedia.mockImplementation(() => useUploadMediaResult);
+
+    const useUploadVideoFrameResult = {
+      uploadVideoFrame: jest.fn(),
+    };
+    useUploadVideoFrame.mockImplementation(() => useUploadVideoFrameResult);
+  });
+
+  it('resetWithFetch calls getMedia with cacheBust:true and then fetchMediaSuccess', async () => {
+    // Set up and make initial call to useContextValueProvider (which calls
+    // getMedia and fetchMediaSuccess once). Sets an empty media state.
+    const getMedia = jest.fn(() =>
```

Comment:
I think this can be written as:
```
const getMedia = jest.fn(async () => ({
  data: [],
  headers: GET_MEDIA_RESPONSE_HEADER,
}));
```

Reply:
Linter doesn't seem to like this "async arrow function with no await expression". So I'll just leave it as it is.

Reply:
Makes sense